app/templates/sidebar.html

```diff
@@ -56,18 +56,23 @@
         </nav>
         <div class="{{ 'collapse-show' if request.path in ['/adminLogs','/manageServiceLearning','/admin','/courseManagement','/manageBonner'] else 'collapse'}}" id="navbarToggleExternalContent">
           <div class="bg-dark ps-3">
-            <a href="/adminLogs" class="nav-link text-white {{ 'active' if request.path =='/adminLogs' }}">
-              Activity Logs
-            </a>
+            {% if g.current_user.isCeltsAdmin and not g.current_user.isCeltsStudentAdmin %}
```

Comment:
those two things are never the same. So the first expression is the only one that matters.